### DISCUSSION TITLE
New Universal Components

### DISCUSSION THREAD

**Discussion by arpit-bhalla** [react-native-elements/react-native-elements#3382] [Ideas]
We have few new Universal component proposals

- [ ] https://github.com/react-native-elements/react-native-elements-universe/issues/3 @knmn2000 
- [ ] https://github.com/react-native-elements/react-native-elements-universe/issues/11
- [ ] https://github.com/react-native-elements/react-native-elements-universe/issues/4 @arpitBhalla 
- [x] ~https://github.com/react-native-elements/react-native-elements-universe/issues/5~ @arpitBhalla 
- [ ] https://github.com/react-native-elements/react-native-elements-universe/issues/7
- [ ] Stepper UI
- [ ] Menu Component

**Comment by knmn2000** [comment]
[This issue](https://github.com/react-native-elements/react-native-elements-universe/issues/3) proposes using the [react-native-table-component](https://www.npmjs.com/package/react-native-table-component) library. 
Would it be okay to add this package as a dependency for RNE, given that the last update to this package was 2 years ago?

**Reply by hsavit1** [reply]
React native web projects really could use an advanced table component. However that library looks like it's no longer maintained

**Reply by KrisLau** [reply]
This is the package I currently use for draggable flatlist: https://github.com/computerjazz/react-native-draggable-flatlist. There's also this tutorial: https://www.youtube.com/watch?v=-39OEXk_mWc. Hopefully that helps as a useful reference.

**Comment by knmn2000** [comment]
I'm working on the first proposal, would it be okay to make a work-in-progress PR?
